sotodlib/site_pipeline/update_obsdb.py

```diff
@@ -335,7 +335,8 @@ def main(config: str,
             logger.info(f"Finished {obs_id} in {time.time()-t1} s")
         else:
             bookcart.remove(bookpath)
-
+    if bad_book_counter !=0 :
+        logger.info(f"Found {bad_book_counter} new bad books, There are now {len(config_dict['known_bad_books'])} known bad books.")
```

Comment:
Fix spacing `if bad_book_counter != 0:`

Have this block raise an error, to satisfy #1195 

Then this should be good to go.